src/main/webapp/data/developers.json

```diff
@@ -62,6 +62,10 @@
       "name": "Adrian Siwiec",
       "username": "pierreSquared"
     },
+    {
+      "name": "AdrianSiwiec",
+      "username": "AdrianSiwiec"
```

Comment:
Looks like this is the same person as the entry just before this?

Reply:
Weird as `https://github.com/pierreSquared` doesn't exist.
Will update accordingly.

Reply:
That means the person changed his/her GitHub username. It's not uncommon. We can't catch all cases unless we explicitly look for them, but if we happen to catch one, no harm to fix it.